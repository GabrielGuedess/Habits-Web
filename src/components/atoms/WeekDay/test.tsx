import { render } from '@testing-library/react';

import { WeekDay } from '.';

describe('<WeekDay />', () => {
  it('should render the heading', () => {
    const { container } = render(<WeekDay weekDay="D" />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
