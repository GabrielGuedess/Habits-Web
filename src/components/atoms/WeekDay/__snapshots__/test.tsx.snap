// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WeekDay /> should render the heading 1`] = `
<div
  class="flex h-10 w-10 items-center justify-center text-xl font-bold text-zinc-400"
>
  D
</div>
`;
